pom.xml

```diff
@@ -134,6 +134,27 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-minikdc</artifactId>
+            <version>${hadoop.version}</version>
+            <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <!-- exclude this from hadoop minikdc as the minikdc depends on
+                    the apacheds-jdbm1 bundle, which is not available in maven central-->
+                    <groupId>org.apache.directory.server</groupId>
+                    <artifactId>apacheds-jdbm1</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <!-- add this to satisfy the dependency requirement of jdbm-->
+            <groupId>org.apache.directory.server</groupId>
+            <artifactId>apacheds-jdbm</artifactId>
+            <version>2.0.0-M2</version>
+            <scope>test</scope>
```

Comment:
This doesn't seem to be working for me -- it's still complaining about not being able to find the bundle artifact. What version of maven do you have? Mine is `Apache Maven 3.2.5 (12a6b3acb947671f09b81f49094c53f426d8cea1; 2014-12-14T09:29:23-08:00)`. Not sure what else would cause there to be a different since everything else is done via maven plugins.